### PR TITLE
chore: remove undefined badge

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**
Remove undefined badge from README

**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
Ya
**Summary**

Removes old badge

**Does this PR introduce a breaking change?**
No

**Other information**
🎄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,6 @@
 [![npm][npm]][npm-url]
 [![Build Status][build-status]][build-status-url]
 [![codecov][codecov-badge]][codecov-url]
-[![Dependencies][deps]][deps-url]
```
Comment (snitin315) [inline README.md:17]: Also need to remove these links `deps` and `deps-url` from the bottom.

Comment (evenstensberg) [reply README.md:17]: Could you fix that?

**Review by snitin315 — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Comment by alexander-akait** [comment]
I think better to fix URL on dependencies than remove them...

**Comment by snitin315** [comment]
Fixed in master.

**Pull request closed without merging**